changelogs/fragments/305-launchd-module-for-macos.yaml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - launchd - new module to control launchd services on macOS hosts (https://github.com/ansible-collections/community.general/pull/305).
```

```suggestion
---
minor_changes:
  - launchd - new module to control launchd services on macOS hosts (https://github.com/ansible-collections/community.general/pull/305).
```
we don't need a changelog fragment for new modules. Please, remove the fragment

---

OK